CMakeLists.txt

```diff
@@ -113,19 +113,24 @@ if(HUNTER_ENABLED)
   hunter_add_package(Protobuf)
   find_package(Protobuf ${hunter_config} REQUIRED)
   list(APPEND package_deps Protobuf)
-  set(protoc_exe protobuf::protoc)
-else()
-  find_program(protoc_exe protoc)
-  if(NOT protoc_exe)
-    message(FATAL_ERROR
-            "Cannot find protoc executable, please install Protobuf")
-  endif()
 endif()
 
 hunter_add_package(protobuf-c)
 find_package(protobuf-c ${hunter_config} REQUIRED)
 list(APPEND package_deps protobuf-c)
 
+if(TARGET protobuf-c::protoc-gen-c AND TARGET protobuf::protoc)
+  message(STATUS "using protoc and protoc-gen-c")
+  set(protoc_exe $<TARGET_FILE:protobuf::protoc>)
+  set(protoc_plugin_arg --plugin=$<TARGET_FILE:protobuf-c::protoc-gen-c>)
+elseif(TARGET protobuf-c::protoc-c)
```

Comment:
I can fix this issue by using `protoc-c` directly. I found that I had issues doing this with the Hunter version of `protoc-c`, so I used an invocation of `protoc` that loaded the `protoc-gen-c` plugin. If Hunter is disabled, I can try using `protoc-c` directly instead.

Reply:
That's what I did in an earlier revision, but there turns out to be no point. protobuf-c 1.3 adds `protobuf-gen-c` and is also the oldest version that supports "proto3". So in practice there's a hard requirement on 1.3 and you can just rely on doing it this way.

It's silly that Hunter doesn't expose a `protobuf-c::protoc-c` target, but in this case it wouldn't help us.